navigation/app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/RerouteActivity.java

```diff
@@ -125,6 +125,11 @@ public void onProgressChange(Location location, RouteProgress routeProgress) {
 
   @Override
   public void onResponse(Call<DirectionsResponse> call, Response<DirectionsResponse> response) {
+    if (response.body().getRoutes() == null) {
```

Comment:
CI still giving a hard time -- Looks like you're going to do a null check on the `body()` too